src/components/ftbot/PairlistConfigActions.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="d-flex flex-row mb-2 gap-2">
+    <b-button
+      title="Delete config"
+      :disabled="!pairlistStore.isSavedConfig"
+      variant="dark"
+      @click="pairlistStore.deleteConfig()"
+      ><i-mdi-delete width="24" height="24"
+    /></b-button>
+
+    <b-form-input v-model="configName" placeholder="Configuration name..." />
+
+    <b-button-group>
+      <b-dropdown variant="dark" title="Saved configs">
+        <b-dropdown-item-button
+          v-for="config in pairlistStore.savedConfigs"
+          :key="config.name"
+          @click="pairlistStore.selectConfig(config)"
+          >{{ config.name }}</b-dropdown-item-button
+        >
+      </b-dropdown>
+      <b-button
+        title="Save config"
+        variant="dark"
+        :disabled="!configName || pairlistStore.config.pairlists.length == 0"
+        @click="pairlistStore.saveConfig(configName)"
+        ><i-mdi-content-save width="24" height="24"
+      /></b-button>
+    </b-button-group>
+    <b-button
+      title="Clone config"
+      variant="dark"
+      :disabled="!pairlistStore.config.name || !pairlistStore.isSavedConfig"
+      @click="pairlistStore.cloneConfig()"
+      ><i-mdi-content-copy width="24" height="24"
+    /></b-button>
+    <b-button
+      title="New config"
+      variant="dark"
+      :disabled="pairlistStore.config.pairlists.length == 0"
+      @click="pairlistStore.newConfig()"
+      ><i-mdi-plus width="24" height="24"
+    /></b-button>
```

Comment:
This should use the editValue view, so it's consistent throughout the UI.
We can add a "clone" button to that view - but i don't see the necessity to have the same functionality twice, with slightly different UI.

That's very bad for UX.

Reply:
As a side note - using the editValue view will also fix the odd coloring in bright mode :) 

![image](https://github.com/freqtrade/frequi/assets/5024695/d0ea6e20-33fc-4247-b835-7415006866bc)